pkg/sinks/new_relic_infra.go

```diff
@@ -110,12 +131,76 @@ type newRelicInfraSink struct {
 	metrics        newRelicInfraSinkMetrics
 }
 
+// HandleObject sends the descriptions for the object to the New Relic Agent
+func (ns *newRelicInfraSink) HandleObject(kubeObj common.KubeObject) error {
+	defer ns.sdkIntegration.Clear()
+
+	common.K8SObjSetGVK(kubeObj.Obj)
```

Comment:
Nit: Super minor, but does the GVK need to be set on the obj for any of the other operations below? 
Otherwise you could just make this `GetGVK` instead, and you wouldn't have to modify the runtime obj.

Reply:
The yaml printers need the GVK set, so this is a leftover from that.
I can check if the describe tooling relies on it or not, else the Kind is only used in this func below :)